### PR TITLE
[Docs-2.0] FAQ-UseWasabi small update 3

### PR DESCRIPTION
**Pull request by BTCparadigm**
Part of a series of PRs required to update FAQ-UsingWasabi

Is line 414 still true?
"On TestNet and RegTest Wasabi uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'`."

r_5519388_LBgQp

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -368,17 +369,18 @@ It is important that you avoid sending the same address to several different ind
...
 :::
 ::::
 
 :::details
 ### Why does Wasabi only use SegWit bech32 addresses?
 
-Wasabi was created after the activation of SegWit, and it makes sense to support the most advanced address type, which has numerous benefits.
-For example, due to the malleability fix of SegWit, you can now remix CoinJoin outputs which are currently unconfirmed.
+Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits.
+For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:381]:
```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```
```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```
```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```

Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:381]: in 2.0 only confirmed coins can coinjoin

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -391,21 +393,26 @@ To check Bech32 adoption and wallets/exchanges support you can follow the [Bitco
 :::
 
 :::details
-### Where can I find my address QR code and public key?
+### Where can I find my address and the corresponding QR code?
 
-You can see the address QR code, public key and the key path in the drop down menu of the `Receive` tab.
-Click on the arrow on the left side of the address to reveal the extra details.
-You can save the png file of the QR code in the right click menu.
+You can see the address and its QR code when you create it through `Receive` tab.
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:398]:
```suggestion
You can see the address and its QR code when you create it through the `Receive` dialog.
```

**Review by MarnixCroes — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -344,12 +344,13 @@ or:
 :::details
 ### How can I change the observers of my receive address?
 
-You can change the observers of your receive address in the right click menu by clicking `Change Observers`, then type in the new label.
-This is useful when you have generated a receiving address with a specific label, but then the cause for receiving has changed.
+You can change the observers of your receive address, as long as it hasn't received any coins yet.
+This can be done through `Reveive->Unused Receive Addresses->Edit`, in the right side of the label.
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:348]:
```suggestion
Go to the Unused Receive Addresses list:  `Receive->Unused Receive Addresses`, and click on the pencil icon to edit the labels.
```

**Review by MarnixCroes — APPROVED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 4]
```diff
@@ -391,21 +391,26 @@ To check Bech32 adoption and wallets/exchanges support you can follow the [Bitco
...
+![Wasabi Wallet address' QR code](/UnusedReceiveAddresses.png "Wasabi Wallet address' QR code")
+
+Currently, there's no way to see used addresses and their QR codes through the GUI.
 :::
 
 :::details
 ### What derivation paths does Wasabi use?
 
 Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki), so the main path is `m/84'/0'/0'`.
 On TestNet and RegTest Wasabi uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'`.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:412]:
```suggestion
On TestNet Wasabi uses the path `m/84'/1'/0'`.
On RegTest it uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'` path.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 5]
```diff
@@ -344,12 +344,13 @@ or:
 :::details
 ### How can I change the observers of my receive address?
 
-You can change the observers of your receive address in the right click menu by clicking `Change Observers`, then type in the new label.
-This is useful when you have generated a receiving address with a specific label, but then the cause for receiving has changed.
+You can change the observers of your receive address, as long as it hasn't received any coins yet.
+Go to the Unused Receive Addresses list:  `Receive->Unused Receive Addresses`, and click on the pencil icon to edit the labels.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:348]:
```suggestion
Go to the Unused Receive Addresses list:  `Receive` -> `Unused Receive Addresses`, and click on the pencil icon to edit the labels.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 6]
```diff
@@ -368,17 +369,16 @@ It is important that you avoid sending the same address to several different ind
...
 :::
 ::::
 
 :::details
 ### Why does Wasabi only use SegWit bech32 addresses?
 
-Wasabi was created after the activation of SegWit, and it makes sense to support the most advanced address type, which has numerous benefits.
-For example, due to the malleability fix of SegWit, you can now remix CoinJoin outputs which are currently unconfirmed.
-There are also large savings on mining fees for SegWit transactions.
+Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:380]:
```suggestion
Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits, like large savings on mining fees for SegWit transactions.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 7]
```diff
@@ -391,21 +391,27 @@ To check Bech32 adoption and wallets/exchanges support you can follow the [Bitco
...
+### Where can I find my address and the corresponding QR code?
 
-You can see the address QR code, public key and the key path in the drop down menu of the `Receive` tab.
-Click on the arrow on the left side of the address to reveal the extra details.
-You can save the png file of the QR code in the right click menu.
+You can see the address and its QR code when you create it through the `Receive` dialog.
+To save the png file of the QR code, hover your mouse over the QR code and click `Save` button that appears to the right side.
 
-![Wasabi Wallet address' QR code](/ReceiveAddressDropDownMenu.png "Wasabi Wallet address' QR code")
+![Wasabi Wallet address' QR code](/ReceiveAddress.png "Wasabi Wallet address' QR code")
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:399]: There is no such a file `ReceiveAddress.png`

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by MarnixCroes** [comment]
can we merge this?

File: docs/FAQ/FAQ-UseWasabi.md (hunk 7 above)
Comment (BTCparadigm) [reply docs/FAQ/FAQ-UseWasabi.md:399]: True. Its in https://github.com/zkSNACKs/WasabiDoc/pull/1088/files

**Comment by yahiheb** [comment]
@BTCparadigm Make a new PR to the master branch

**Pull request closed without merging**